src/lib/utils.test.ts

```diff
@@ -215,17 +221,19 @@ test('objectsHaveShallowEquality', () => {
 })
 
 test('decodeBase64', () => {
-  jsc.assertForall(jsc.array(jsc.uint8), byteArray => {
-    let binaryString = ''
-    for (let byte of byteArray) {
-      binaryString += String.fromCharCode(byte)
-    }
-    const b64string = btoa(binaryString)
-    const decoded = decodeBase64(b64string)
-
-    expect(Uint8Array.from(byteArray)).toEqual(decoded)
-
-    // If the above expect(...) assertion fails, we won't reach here.
-    return true
-  })
+  fc.assert(
+    fc.property(fc.array(fc.integer({min: 0, max: 255})), byteArray => {
+      let binaryString = ''
+      for (let byte of byteArray) {
+        binaryString += String.fromCharCode(byte)
+      }
+      const b64string = btoa(binaryString)
+      const decoded = decodeBase64(b64string)
+
+      expect(Uint8Array.from(byteArray)).toEqual(decoded)
+
+      // If the above expect(...) assertion fails, we won't reach here.
+      return true
```

Comment:
No need for return with fast-check. No return or return true is success. Throw or return false is failure.